Postman Test Suite/Ed-Fi ODS-API ChangeQueries Key Changes and Deletes Test Suite.postman_collection.json

```diff
@@ -10571,7 +10571,7 @@
 											}
 										},
 										"url": {
-											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/staffSectionassociations/keyChanges?minChangeVersion={{known:{{scenarioId}}:initialChangeVersion}}",
+											"raw": "{{ApiBaseUrl}}/data/v3/ed-fi/staffSectionassociations/keyChanges?minChangeVersion={{known:{{scenarioId}}:newestChangeVersion}}",
```

Comment:
The `{{known:{{scenarioId}}:initialChangeVersion}}` variable doesn't exist, so it was sending an invalid filter all along.
The API continued processing the request, so the test seemed to be working as expected. After my change, the API detected the invalid parameter and started returning HTTP 400.
So, I updated the test to reflect what seemed to be the original intention.